packages/graphql-language-service-interface/src/__tests__/getDefinition-test.ts

```diff
@@ -43,6 +40,10 @@ describe('getDefinition', () => {
           },
         ],
       );
+<<<<<<< HEAD:packages/graphql-language-service-interface/src/__tests__/getDefinition-test.js
+=======
+
+>>>>>>> chore: Convert LSP tests to jest from mocha (#871):packages/graphql-language-service-interface/src/__tests__/getDefinition-test.ts
```

Comment:
Merge conflict

Reply:
thanks, theres some cleanup as i started on this before we merged the lsp jest conversion